cfme/infrastructure/virtual_machines.py

```diff
@@ -706,26 +706,29 @@ def power_control_from_provider(self, option):
             raise OptionNotAvailable(option + " is not a supported action")
 
     def migrate_vm(self, email=None, first_name=None, last_name=None,
-                   host_name=None, datastore_name=None):
+                   host=None, datastore=None):
         view = navigate_to(self, 'Migrate')
         first_name = first_name or fauxfactory.gen_alphanumeric()
         last_name = last_name or fauxfactory.gen_alphanumeric()
         email = email or "{}@{}.test".format(first_name, last_name)
         try:
             prov_data = cfme_data["management_systems"][self.provider.key]["provisioning"]
+            host_name = host or prov_data.get("host")
+            datastore_name = datastore or prov_data.get("datastore")
         except (KeyError, IndexError):
             raise ValueError("You have to specify the correct options in cfme_data.yaml")
-        provisioning_data = {
+        request_data = {
             'request': {
                 'email': email,
                 'first_name': first_name,
-                'last_name': last_name},
-            'environment': {"host_name": {'name': prov_data.get("host")}},
+                'last_name': last_name,
+            },
+            'environment': {
+                'host_name': {'Name': host_name},
```

Comment:
QUESTION: are you sure that upper case name is correct ?

NB: please use one type of quotation marks.

Reply:
FOr the upper case, it works this way. 
I will change the quotes.